en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -143,7 +143,7 @@ Follow the instructions below to set up and configure the databases for the WSO2
 
 ## Step 4 - Configure WSO2 IS with WSO2 API-M
 
-1. Download the [WSO2 IS Connector]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.16.zip).
+1. Download the [WSO2 IS Connector]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.17.zip).
 
 2. Extract the distribution and copy the following JAR files, which are in the `<wso2is-extensions-1.0.16.1>/dropins` directory, to the `<IS_HOME>/repository/components/dropins` directory.
```

Comment:
@isharac , looks like the versions mentioned in step 2 will have to change as well.